client/settings/express-checkout/index.js

```diff
@@ -176,6 +200,29 @@ const ExpressCheckout = () => {
 							</a>
 						</div>
 					</li>
+					{ displayLinkPaymentMethod && (
```

Comment:
If I get it right, when StripeLink is not gated for a platform account, we still can see the widget?

Reply:
indeed, we should check also if StripeLink is gated . I will add the changes.